packages/composables/src/useLandingSearch.test.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, expect, it, vi } from "vitest";
+import { useLandingSearch } from "./useLandingSearch";
+import { shallowMount } from "@vue/test-utils";
+import * as apiExports from "@shopware-pwa/api-client";
+import LandingPageMock from "./mocks/LandingPage";
+
+const Component = {
+  template: "<div/>",
+  props: {},
+  setup() {
+    const { search } = useLandingSearch();
+    return {
+      search,
+    };
+  },
+};
+
+const getMockProvide = () => ({
+  global: {
+    provide: {
+      shopware: {
+        apiInstance: {
+          config: {},
+        },
+      },
+    },
+  },
+});
+
+describe("useLandingSearch", () => {
+  const wrapper = shallowMount(Component, getMockProvide());
+
+  vi.spyOn(apiExports, "getLandingPage").mockImplementation(() => {
+    return new Promise((resolve) => {
+      resolve(LandingPageMock);
+    });
+  });
```

Comment:
less complicated:
```suggestion
  vi.spyOn(apiExports, "getLandingPage").mockImplementation(async () => {
    return LandingPageMock;
  });
```

please make sure TS is not showing any problems with mock type incompatibility